wazo_chatd/plugins/teams_presence/services.py

```diff
@@ -98,14 +98,18 @@ def update_presence(self, payload: dict, user_uuid: str):
             tenant_uuids = [self._synchronizers[user_uuid].tenant_uuid]
             presence = subscription['resource_data']
             state = PRESENCES_MAP.get(presence['availability'])
+
+            if state == 'invisible':
+                logger.debug(
+                    'discarding offline presence update for user `%s`', user_uuid
+                )
+                return
```

Comment:
If I understand correctly, a user being `Available` on team and disconnecting its client, will stay `Available`?